test/extensions/common/dynamic_forward_proxy/dns_cache_impl_test.cc

```diff
@@ -129,34 +131,43 @@ void verifyCaresDnsConfigAndUnpack(
 
 TEST_F(DnsCacheImplTest, PreresolveSuccess) {
   Network::DnsResolver::ResolveCb resolve_cb;
-  std::string hostname = "bar.baz.com:443";
-  EXPECT_CALL(*resolver_, resolve("bar.baz.com", _, _))
+  std::string host = "bar.baz.com";
+  uint32_t port = 443;
+  std::string authority = host + ":" + std::to_string(port);
```

Comment:
nit: I think absl::StrCat() is the canonical way to do this sort of concatenation.